Code_of_conduct.md

```diff
@@ -0,0 +1,94 @@
+# NodejsSecurify Project Code of Conduct
+
+## Welcome
+
+Welcome to the **NodejsSecurify** repository! **NodejsSecurify** is an advanced (White Box Testing Automation) npm package designed to enhance the security of Node.js applications. It provides a comprehensive set of security features and analysis capabilities to identify potential vulnerabilities and enforce best practices in accordance with OWASP guidelines. Our goal is to create an open, inclusive, and respectful environment where developers can engage and collaborate. We are committed to maintaining a harassment-free experience for everyone involved.
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in **NodejsSecurify** a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+We pledge to act in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community, ensuring that all contributors feel supported and empowered to engage meaningfully in the project.
+
+## Our Standards
+
+To maintain a positive, constructive, and respectful environment in the **NodejsSecurify** project, all participants are expected to adhere to the following standards:
+
+### Positive Behavior:
+- **Respectful Communication**: Engage with others respectfully, ensuring that discussions remain professional and focused on the goals of the project.
+- **Constructive Feedback**: Provide feedback that is constructive, helpful, and aimed at improving contributions while being respectful and encouraging.
+- **Collaboration**: Work openly and cooperatively with others, sharing knowledge and ideas that contribute to the growth of the platform.
+- **Inclusivity**: Use language and actions that welcome and include all participants, regardless of their background or experience level.
+- **Recognition**: Acknowledge and appreciate the contributions of all members, whether through code, design, feedback, or discussions.
+- **Professionalism**: Maintain professionalism in all interactions, ensuring that the project’s goals and values are prioritized.
+
+### Unacceptable Behavior:
+- **Harassment**: Any form of harassment, including unwelcome comments, personal attacks, or inappropriate behavior, is strictly prohibited.
```

Comment:
change "is strictly prohibited" to "is not permitted" or "is strictly discouraged."